src/Logging/src/SerilogDynamicLoggerCore/ISerilogOptions.cs

```diff
@@ -2,18 +2,24 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Collections.Generic;
 
 namespace Steeltoe.Extensions.Logging.SerilogDynamicLogger
 {
     public interface ISerilogOptions
     {
-       string ConfigPath { get; }
+        public interface ISerilogOptions
+        {
+            string ConfigPath { get; }
 
-       MinimumLevel MinimumLevel { get; set; }
+            MinimumLevel MinimumLevel { get; set; }
 
-       IEnumerable<string> SubloggerConfigKeyExclusions { get; set; }
+            [Obsolete("No longer needed with current implementation. Will be removed in next major release")]
```

Comment:
This message could be confusing since it wasn't removed in 3.0

Reply:
fixed.